packages/expect-puppeteer/src/matchers/toFill.js

```diff
@@ -3,7 +3,9 @@ import toMatchElement from './toMatchElement'
 async function toFill(instance, selector, value, options) {
   const element = await toMatchElement(instance, selector, options)
```

Comment:
Actually, `toMatchElement` will receive a delay option that is not required. Can you please avoid this?

```js
const { delay, ...toMatchElementOptions } = options || {}
```

Reply:
done in 84841d1